CHANGES.rst

```diff
@@ -1,8 +1,8 @@
 CHANGES
 =======
 
-2.3.9 (unreleased)
-------------------
+2.4 (unreleased)
```

Comment:
Ah, ok, I would say lets keep get_portal then and go on with a 2.4.x for Plone 5.1+ and branch away a 2.3.x